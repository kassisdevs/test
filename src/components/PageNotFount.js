import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFount = () => (
  <div>404! <Link to="/" >Home</Link></div>
);

export default PageNotFount;