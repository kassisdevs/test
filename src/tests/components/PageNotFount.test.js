import React from 'react';
import { shallow } from 'enzyme';
import PageNotFount from '../../components/PageNotFount';

test('Should render help page', () => {
  const wrapper = shallow(<PageNotFount />);
  expect(wrapper).toMatchSnapshot();
});