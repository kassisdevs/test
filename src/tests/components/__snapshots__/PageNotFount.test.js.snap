// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render help page 1`] = `
<div>
  404! 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
